taigit/src/libraries/Taiga.ts

```diff
@@ -877,4 +877,29 @@ check_for_retrospective(project_id : number, sprint_id : number) : Promise<wiki_
         }
     }
     return retro_page;
-}
\ No newline at end of file
+}
+
+
+
+
+/**
+ * @summary This call return list of sprint velocity (unit points) based on project Id
+ * @param projectId the ID for the access of project
+ * @returns
+ *  * Array of [sprint_end, velocity]
+ * * {
+ *          sprint_end : boolen // if sprint end currently
+ *          velocity : number //sprint velocity
+ * }
+ */
+export async function
+proj_spvelocity(projId : number) : Promise<Array<[boolean, number]>> {
+    //let data = (await axios.get("https://api.taiga.io/api/v1/milestones/"+sprintId.toString()+ '/stats')).data;
+    let sp_list =  (await axios.get(`https://api.taiga.io/api/v1/milestones?project=${projId}`)).data;
+    let output : Array<[boolean, number]> = [];
+    for(let sp of sp_list){
+        let sp_velocity = (await sprint_velocity_pts(sp.id));
+        output.push(sp_velocity);
+    }
+    return output.reverse();
+}
```

Comment:
I don't think any guarantees as to the order of the objects can be made. You should definitely put each sprint ID with the returned sprint velocity objects to differentiate each other. 

you should also return objects instead of heterogenous arrays of values.

Reply:
"I don't think any guarantees as to the order of the objects can be made." Not understand what you meaning? 
My current version works fine. I do not want to change it. Change return to object, will need adding a interface to explained data details, so that I can use the object in other API calls. Also, I do not think sprint ID is required as when you call this function you need provide sprint ID. Sprint velocity is a statistical value, its further usage do not care which sprint it belongs if you check the book.